src/components/TodoItem.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Todo, User } from '../interfaces';
+
+interface Props {
+  todo: Todo;
+  user: User;
+}
+
+const TodoItem: React.FC<Props> = ({ todo, user }) => {
+  return (
+    <li className={todo.completed ? 'completed' : ''}>
```

Comment:
```suggestion
    <li className={todo.completed ? 'completed' : ''}>
```
используй библиотеку класс найм, не должно быть пустых классов